postgres/postgres_test.go

```diff
@@ -1,23 +1,25 @@
 package postgres_test
 
 import (
+	"bytes"
 	"context"
+	"encoding/json"
 	"fmt"
-	"os"
 	"testing"
 	"time"
 
 	"github.com/jarri-abidi/vehicle-tracking/karma"
 	"github.com/jarri-abidi/vehicle-tracking/postgres"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 type optTrips func(trips *[]karma.Trip)
 
-func generateTrips(opts ...optTrips) []karma.Trip {
+func generateTrips(tripsCount int32, opts ...optTrips) []karma.Trip {
```

Comment:
```suggestion
func generateTrips(count int32, opts ...optTrips) []karma.Trip {
```
var names should be context-aware.. otherwise even for opts, we would name it `optTrips ...optTrips` but it's not needed because it's obvious everything here is about trips